hardhat.config.ts

```diff
@@ -63,45 +43,53 @@ const config: HardhatUserConfig = {
   networks: {
     mainnet: {
       chainId: 1,
-      url: process.env.MAINNET_PROVIDER || "",
+      url:
+        process.env.MAINNET_PROVIDER || "https://ethereum-rpc.publicnode.com",
       accounts: process.env.MAINNET_DEPLOYER_PRIVATE_KEY
         ? [process.env.MAINNET_DEPLOYER_PRIVATE_KEY]
-        : [],
+        : [dummyPrivateKey],
```

Comment:
Whats this for?

Reply:
I'm assuming its for testing. Surprised the test environment can't pick up on and deploy with Account 0 in the hardhat node

Reply:
Yeah, kinda goofy.

This is specifically so that the `hardhat verify` task doesn't throw an error. For whatever reason, that task doesn't work unless all of the provider URLs and private keys in this config file are valid values. So, in the common case where you're only putting private keys in your `.env` as quick temporary measures for actually deploying, the task won't run.